website/source/docs/providers/aws/d/iam_policy_document.html.markdown

```diff
@@ -71,14 +72,15 @@ valid to use literal JSON strings within your configuration, or to use the
 
 The following arguments are supported:
 
-* `id` (Optional) - An ID for the policy document.
+* `version` (Optional) - An policy version. Defaults to `2012-10-17`.
```

Comment:
@apparentlymart the reason I had was for existing ones what shouldn't be modified (for whatever reason) - happy to point out in the docs that the default should be fine for almost all use cases though